Python/Python.sublime-syntax

```diff
@@ -956,11 +945,30 @@ contexts:
         1: punctuation.section.sequence.begin.python
         2: punctuation.section.sequence.end.python
       push: after-expression
-    - match: \((?={{simple_expression}},|\s*\*{{path}})
+    - match: \((?=\s*\*{{path}})
```

Comment:
Can someone elaborate on `{{path}}`? Removing this whole rule doesn't break any tests. Includes use their own contexts, so I am not sure what this pattern is finally used for. May it be removed?

Reply:
I'm also not sure what it is used for. As it breaks no syntax tests to remove it, maybe let's do so. We can always reintroduce it later with an appropriate test assertion if it turns out to be necessary.

Reply:
I am under the impression it had to do with import statements somehow. Those have been using their own lists for a while now.

Reply:
It was introduced in https://github.com/sublimehq/Packages/pull/999/files#diff-3038ea28e1ccdd590636eb111ab962d610e4e9cae9498f1dd9a852dc347a2d55R890 - dict and set had the same `{{path}}` logic which got relaxed in https://github.com/sublimehq/Packages/pull/2174/files

Reply:
As imports already used their own lists at this point I can't imagine any use case in which a path would point to a tuple rather than a group. Anyway, as a `,` being present within parentheses is used to indicate a tuple, which also works in case it appears on a subsequent line, I don't think this path-heuristic is still useful.

I just think of old format strings such has `"foo: %s" % (var,)` or even tuple initializations such as `var = (1,)`. Python only expects a comma to be present to recognize them as tuple.

The only one who can tell for sure is @FichteFoll as he added this heuristics, but I don't think we need it any longer.

Reply:
I don't think we need it either. I know I added it 4 years ago, but I couldn't tell you why. 

Syntatically, the only thing that matters is whether you have a single- or a double-star expansion, since that distinguishes the set or dict literal. Requiring a `{{path}}` to match there is unnecessarily restrictive.

Reply:
Actually, this is purely theoretical and irrelevant to begin with since you cannot start a tuple with a starred expression.

```
In [1]: (*range(1, 5))
  File "<ipython-input-1-b52559586778>", line 1
    (*range(1, 5))
     ^
SyntaxError: can't use starred expression here
```

Reply:
removed it :)